game-core/src/main/java/games/strategy/triplea/delegate/BasePersistentDelegate.java

```diff
@@ -10,14 +10,6 @@
  * etc.
  * Persistent Delegates are active all the time.
  */
+@SuppressWarnings("AbstractClassWithOnlyOneDirectInheritor")
```

Comment:
The warning here made me look at collapsing this class with its child; eventually I read the javadoc comment and backed up a few steps. Suppression is in place given the commentary states this situation is explicit.